zrml/prediction-markets/src/tests.rs

```diff
@@ -63,6 +63,30 @@ fn gen_metadata(byte: u8) -> MultiHash {
     MultiHash::Sha3_384(metadata)
 }
 
+fn reserve_sentinel_amounts() {
+    // Reserve a sentinel amount to check that we don't unreserve too much.
+    assert_ok!(Balances::reserve_named(&PredictionMarkets::reserve_id(), &ALICE, SENTINEL_AMOUNT));
+    assert_ok!(Balances::reserve_named(&PredictionMarkets::reserve_id(), &BOB, SENTINEL_AMOUNT));
+    assert_ok!(Balances::reserve_named(
+        &PredictionMarkets::reserve_id(),
+        &CHARLIE,
+        SENTINEL_AMOUNT
+    ));
+    assert_ok!(Balances::reserve_named(&PredictionMarkets::reserve_id(), &DAVE, SENTINEL_AMOUNT));
+    assert_ok!(Balances::reserve_named(&PredictionMarkets::reserve_id(), &EVE, SENTINEL_AMOUNT));
+    assert_ok!(Balances::reserve_named(&PredictionMarkets::reserve_id(), &FRED, SENTINEL_AMOUNT));
+    assert_eq!(Balances::reserved_balance(&ALICE), SENTINEL_AMOUNT);
+    assert_eq!(Balances::reserved_balance(&BOB), SENTINEL_AMOUNT);
+    assert_eq!(Balances::reserved_balance(&CHARLIE), SENTINEL_AMOUNT);
+    assert_eq!(Balances::reserved_balance(&DAVE), SENTINEL_AMOUNT);
+    assert_eq!(Balances::reserved_balance(&EVE), SENTINEL_AMOUNT);
+    assert_eq!(Balances::reserved_balance(&FRED), SENTINEL_AMOUNT);
+}
+
+fn check_reserve(account: &AccountIdTest, expected: Balance) {
```

Comment:
Yeah, this is a good solution, because it abstracts away the `SENTINEL_AMOUNT` from the tester (at least up to the point where a test fails...).